starters/cra-rxjs-styled-components/src/components/FetchExample/FetchExample.tsx

```diff
@@ -1,35 +1,52 @@
+import React, { useState, useEffect, useMemo } from 'react';
+import { BehaviorSubject, of, merge, mergeMap } from 'rxjs';
+import { $ } from 'react-rxjs-elements';
+import {
+  debounceTime,
+  map,
+  distinctUntilChanged,
+  filter,
+  switchMap,
+  catchError,
+  startWith,
+} from 'rxjs/operators';
+
 import {
   FetchExampleContainer,
   HeaderContainer,
   Header,
   HomeLinkDiv,
   ReturnHomeLink,
   Message,
+  Loader,
 } from './FetchExample.styles';
-import { useState, useEffect } from 'react';
 import { fromFetch } from 'rxjs/fetch';
 
 export const FetchExample = () => {
-  const [message, setMessage] = useState<string>('');
-  const [loading, setLoading] = useState<boolean>(true);
-
-  useEffect(() => {
-    const subscription = fromFetch(
-      'https://api.starter.dev/hello?greeting=from This Dot Labs!'
-    ).subscribe((response) => response.text().then((data) => setMessage(data)));
-    setLoading(false);
-    return () => subscription.unsubscribe();
-  }, []);
-
-  return (
-    <FetchExampleContainer>
-      <HeaderContainer>
-        <Header>RxJS Fetch Data from API</Header>
-      </HeaderContainer>
-      <Message>Message: {loading ? '...Loading message' : message}</Message>
-      <HomeLinkDiv>
-        <ReturnHomeLink to="/">Return Home</ReturnHomeLink>
-      </HomeLinkDiv>
-    </FetchExampleContainer>
+  const stream$ = useMemo(
+    () =>
+      fromFetch(
+        'https://api.starter.dev/hello?greeting=from This Dot Labs!'
+      ).pipe(
+        mergeMap((response) => response.text()),
+        // now we'll map not only to text
+        // but to JSX
+        map((data) => (
+          <FetchExampleContainer>
+            <HeaderContainer>
+              <Header>RxJS Fetch Data from API</Header>
+            </HeaderContainer>
+            <Message>Message: {data}</Message>
+            <HomeLinkDiv>
+              <ReturnHomeLink to="/">Return Home</ReturnHomeLink>
+            </HomeLinkDiv>
+          </FetchExampleContainer>
+        )),
+        catchError(() => of(<div className="err">ERROR</div>)),
+        startWith(<Loader />)
+      ),
+    []
   );
+
+  return <$>{stream$}</$>;
```

Comment:
The previous component was correct. We want to just replace the `<Message>` with the `Loader`, not the entire component.